satpy/tests/compositor_tests/__init__.py

```diff
@@ -296,6 +296,73 @@ def test_inline_composites(self):
                          ['IR_108', 'IR_087'])
 
 
+class TestColormapCompositor(unittest.TestCase):
+    """Test the ColormapCompositor."""
+
+    def test_build_colormap(self):
+        from satpy.composites import ColormapCompositor
+        import numpy as np
+        import xarray as xr
+        cmap_comp = ColormapCompositor('test_cmap_compositor')
+        palette = np.array([[0, 0, 0], [127, 127, 127], [255, 255, 255]])
+        cmap, sqpal = cmap_comp.build_colormap(palette, np.uint8, {})
+        self.assertTrue(np.allclose(cmap.values, [0, 1]))
+        self.assertTrue(np.allclose(sqpal, palette / 255.0))
+
+        palette = xr.DataArray(np.array([[0, 0, 0], [127, 127, 127], [255, 255, 255]]),
+                               dims=['value', 'band'])
+        palette.attrs['palette_meanings'] = [2, 3, 4]
+        cmap, sqpal = cmap_comp.build_colormap(palette, np.uint8, {})
+        self.assertTrue(np.allclose(cmap.values, [2, 3, 4]))
```

Comment:
You can do `np.testing.assert_allclose(...)` instead of `self.assertTrue(np.allclose(...))`. You may have known that already and prefer the unittest based assert and if so, no problem.

Reply:
I know that, but I usually use unittest's assert to make use of `self` and avoid a style error of not using it.